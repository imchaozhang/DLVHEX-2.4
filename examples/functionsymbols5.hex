p(f(f(f(a)))).
p(Y) :- p(X), &functionDecompose[X, 1](Y), &isFunctionTerm[X]().
