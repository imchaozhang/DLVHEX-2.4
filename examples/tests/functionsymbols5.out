{p(f(f(f(a)))),p(f(f(a))),p(f(a)),p(a)}
